sdcm/remote/remote_long_running.py

```diff
@@ -0,0 +1,44 @@
+import logging
+from uuid import uuid4
+from pathlib import Path
+
+from sdcm.remote.remote_cmd_runner import RemoteCmdRunnerBase
+from sdcm.remote.libssh2_client.result import Result
+from sdcm.remote.libssh2_client.exceptions import UnexpectedExit
+
+logger = logging.getLogger(__name__)
+
+
+def run_long_running_cmd(remoter: RemoteCmdRunnerBase,
+                         cmd: str,
+                         timeout: float | None = None,
+                         ignore_status: bool = False,
+                         verbose: bool = True,
+                         retry: int = 1):
+
+    cmd_uuid = uuid4()
+    cmd_stdout_log = Path('/tmp') / f'remoter_stdout_{cmd_uuid}.log'
+    cmd_stderr_log = Path('/tmp') / f'remoter_stderr_{cmd_uuid}.log'
+    cmd_exit_log = Path('/tmp') / f'remoter_exit_{cmd_uuid}.log'
+    cmd_bash = Path('/tmp') / f'remoter_cmd_{cmd_uuid}.sh'
+
+    remoter.run(f'echo "{cmd}" > {cmd_bash}')
+    remote_command = f'(bash +e +x -o pipefail {cmd_bash} ; echo $? > {cmd_exit_log}) >& {cmd_stdout_log} 2> {cmd_stderr_log} </dev/null  &  jobs -p'
+    if verbose:
+        logger.debug("<%s> execute run_long_running_cmd: %s", remoter.hostname, remote_command)
+    pid = remoter.run(remote_command, retry=0, verbose=verbose).stdout
+
+    remoter.run(f'while [ -e /proc/{pid.strip()} ]; do sleep 0.1; done', timeout=timeout, retry=retry, verbose=False)
```

Comment:
yes the idea that this command could be retried multiple times, and would be limited mostly by timeout

while the actual nodetool command we can't retry

Reply:
but this retry mechanism with this implementation doesn't work correctly - in case of network error (we want to mitigate) while loop will be retried only as per invocation (let's say no retry) - in that case we're not protected.

And from the other hand, if we want to retry x times, then we should watch `exit_code` and make decision based on that

Reply:
now hardcode for 10 retries, if we'll find occasion it might happen more than that we could make it higher